### PR TITLE
fix: get intact username when it contains spaces

### PR DESCRIPTION
**Pull request by wusitee**
Fix the issue #40

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -11,7 +12,12 @@ pub struct Config {
 
 pub fn login() -> Config {
     print!("Username: ");
-    let name = read!();
+    io::stdout().flush().expect("Unable to flush stdout");
+    let mut name = String::new();
+    std::io::stdin()
```
Comment (hey2022) [inline src/config.rs:17]:
```suggestion
    io::stdin()
```

File: src/config.rs
Comment (hey2022) [inline src/config.rs:1]: Should we still use text_io crate or not, it seems to be no longer developed.

Comment (wusitee) [reply src/config.rs:1]: Is there any replacement?

Comment (wusitee) [reply src/config.rs:1]: Or just code our own marcros.

**Review by hey2022 — APPROVED** [review]
LGTM, squash merge.

**Comment by hey2022** [comment]
What happened to squashing?

**Comment by wusitee** [comment]
Sry

**Pull request merged**